### ISSUE TITLE
Grafana loki api " /api/prom/push" is Deprecated and when i am configuring kyverno reporter getting "device or resource busy" error

### ISSUE THREAD

**Issue by vjayanthilal** [fjogeleit/policy-reporter#172]
![image](https://user-images.githubusercontent.com/99331615/180270361-9c9ac359-ef2f-4086-9177-6cc597961be9.png)

i am able to send data to loki with promtail , but kyverno reporter is not able to send the logs

**Comment by fjogeleit** [comment]
Thanks for report, I will have a look on it.

**Comment by fjogeleit** [comment]
I published a new Helm Chart Version 2.10.3 where you can overwrite the API path with the new `target.loki.path` configuration.

```yaml
target:
  loki:
    host: "http://loki...."
    path: "/loki/api/v1/push"
```

If you still have issues let me know and then maybe you can provide some logs from your loki instance. I don't find any information about your error message.
